gate-web/config/gate.yml

```diff
@@ -35,7 +35,8 @@ services:
             packageType: RPM
 # optional services:
   echo:
-    enabled: false
+    baseUrl: http://localhost:8089
+    enabled: true
```

Comment:
Why does this PR modify Echo configurations?

Reply:
I believe it's to improve the local development story ... do these configs get used for anything other than running locally? Does halyard use them @ezimanyi? (we don't use these configs internally so a bit out of the loop).

FWIW `echo` is no longer an optional service as pipeline triggering flows through it.

Reply:
@ajordens : Halyard does not use these configs.  Halyard uses the ones in the `halconfig` directory of each service as a base, which it then templates on the settings generated from your halyard config, but doesn't use the ones in `service-web/config/...`

Reply:
👍 I thought that's how it worked but thanks for clarifying @ezimanyi!

Reply:
When I run Gate without Echo being enabled, and execute a pipeline, I get the following stacktrace:

```
2019-02-19 10:28:35.671 ERROR 5815 --- [nio-8084-exec-4] c.n.s.g.controllers.PipelineController   : Unable to trigger pipeline (application: foo, pipelineId: NewTemplate)

java.lang.NullPointerException: Cannot invoke method postEvent() on null object
        at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91) ~[groovy-all-2.4.15.jar:2.4.15]
        at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:47) ~[groovy-all-2.4.15.jar:2.4.15]
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47) ~[groovy-all-2.4.15.jar:2.4.15]
        at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34) ~[groovy-all-2.4.15.jar:2.4.15]
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47) ~[groovy-all-2.4.15.jar:2.4.15]
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116) ~[groovy-all-2.4.15.jar:2.4.15]
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:128) ~[groovy-all-2.4.15.jar:2.4.15]
        at com.netflix.spinnaker.gate.services.PipelineService.triggerViaEcho(PipelineService.groovy:114) ~[main/:na]
```

I had forgotten this was included in this PR.  If it makes your life easier, I'm happy to submit this as a separate PR, and also move the `echo` config out of the optional block in `gate.yml`

Reply:
Fixing the local dev config seems alright to me.